tools/gluten-it/common/src/main/scala/io/glutenproject/integration/tpc/action/QueriesCompare.scala

```diff
@@ -19,15 +19,26 @@ package io.glutenproject.integration.tpc.action
 import io.glutenproject.integration.stat.RamStat
 import io.glutenproject.integration.tpc.{TpcRunner, TpcSuite}
 
-import org.apache.spark.sql.{SparkSessionSwitcher, TestUtils}
+import org.apache.spark.sql.{QueryRunner, SparkSessionSwitcher, TestUtils}
 
 import org.apache.commons.lang3.exception.ExceptionUtils
 
-case class QueriesCompare(scale: Double, queryIds: Array[String], explain: Boolean, iterations: Int)
+import scala.collection.mutable
+import scala.collection.mutable.ListBuffer
+
+case class QueriesCompare(
```

Comment:
Please add the script that how to generate the plan files, so that developers can re-generate files if tests not passed.